python-blivet.spec

```diff
@@ -25,16 +34,34 @@ Source0: http://github.com/rhinstaller/blivet/archive/%{realname}-%{realversion}
 %global pyudevver 0.18
 
 BuildArch: noarch
-BuildRequires: gettext
-BuildRequires: python3-pocketlint >= %{pocketlintver}
-BuildRequires: python3-devel python3-setuptools
 
 %description
 The python-blivet package is a python module for examining and modifying
 storage configuration.
 
+%package -n %{realname}-data
+Summary: Data for the %{realname} python module.
+
+BuildRequires: systemd
```

Comment:
This is needed for the `%{_unitdir}` macro -- https://fedoraproject.org/wiki/Packaging:Systemd#.25files_section